### DISCUSSION TITLE
Creator - needs to design a financially sustainable tour

### DISCUSSION THREAD

**Discussion by sbaycheng** [pair-cg/general-contributions#72] [User Stories]
# 1. Need:
_What is your need?_

I'm making a new show (music, theatre, dance, media) with my company designed to tour. To ensure that I create a financially sustainable show that can travel as far as possible and be accessible in multiple locations with various technical availabilities, I need to know: which venues (location, spaces) are accepting touring performances; technical capacities of venues; compensation at each (possibly from historical data of past performances). 

# 2. Action:
_What is an action that can be taken towards fulfilling that need?_

- search website/database with filters: venue parameters, cost, technical capacities
- download available list
- cross-reference with other data (e.g., travel restrictions or modes of transport) 

# 3. Desired Outcome:

_What is the desired outcome?_

- clear parameters for a show's ideal touring opportunities, budget, cast size and technical requirements

# 4. Perspective:
_From what perspective is this user story coming from?_

Creator

# 5. Stakeholder Group:
_Which stakeholder group most closely fits the perspective of this User Story?_

Artists, companies, freelancers

# 6. Additional info:
_Here's more information._

**Comment by mariel-marshall** [comment]
Hi @sbaycheng thanks for this! 

1. Can you spell out for us what parameters of the venue affect the cast size?
2. What is the purpose of cross-referencing? Is it to eliminate some of the venues from the list, or is it being used to influence the creative design of the project.

**Reply by sbaycheng** [reply]
1. If I'm trying to make sure that the show is financially successful (or at least has a good chance based on past precedent), I want to know:
 - location of possible venues
 - related expenses (e.g., city & average cost to stay)
 - other cost offsets (e.g., subsidized housing?)
 - proximity to other venues (i.e., travel time)
 - audience capacity at venue: how many tickets can be sold for each show
 - sales history: what percentage of tickets does the venue usually sell? For some types of shows more than others?
 - active subscriber base
 - ticket price range
 - production history: what kind of shows appear there? which ones do well, e.g., sell +75% of the run?

I think I'm probably overlooking some variables, but take the following example: If I have a list of venues that regularly produce Shakespeare and I know the ticket range and for which venues, how many tickets can be sold over the course of the tour, the percentage of sales to expect based on those venues past production history, and the travel costs for the relevant locations (hotel or other boarding costs) then I can determine what the costs of the tour are likely to be per touring cast member. Therefore, if I know that a version of Shakespeare is likely to sell well in venues a, b, and c, and I know the cost of touring to those venues, then I can determine the most efficient size of the cast. If I need to go to more, smaller venues, I might want to keep the cast smaller, especially if I see that the top selling productions for those venues average 5-8 member casts. It would also help to know if one- or two-person shows have done well. Does this help?

(To be clear, this is a pretty crass way of thinking about this process. However, if you look at the success of someone like the US playwright Lauren Gunderson, she has basically built a career of writing plays aimed at very particular kinds of venues and audiences, i.e., regional theatres with particular audience bases and preferences. Although she's very rarely produced in New York, in 2017 and 2019, she was the most produced playwright in the US who was not named Shakespeare: https://slate.com/culture/2019/10/lauren-gunderson-profile-most-popular-playwright-marie-curie.html. 

2. Both. Cross-referencing would allow me to select the optimal tour schedule to minimize travel costs and time. For example, it might be worth travelling further for one kind venue profile (e.g., larger and/or regularly sells out touring shows). Having as much data as possible to coordinate would help plan for the most efficient form of production. 

Also, I have a list of venues and the types of shows that regularly do well there, then I can look at my own process for what elements are most likely to make my show appealing to these venues and successful to audiences.
